static/scss/icomoon.scss

```diff
@@ -3,6 +3,13 @@
   font-family: "icomoon";
   font-weight: normal;
   font-style: normal;
+  src: url("#{$static-path}/fonts/icomoon.eot?dd3z7r"),
```

Comment:
Getting this error in console: 
downloadable font: rejected by sanitizer (font-family: "icomoon" style:normal weight:400 stretch:100 src index:0)

Reply:
Firefox does not support EOTs; it uses WOFF and TTF.

Reply:
here is the reference image
![image](https://user-images.githubusercontent.com/4043989/95978985-64f81400-0e34-11eb-967f-8be20e60d76f.png)

Reply:
If the `icomoon` font had not been applied then icons would have been broken. 

What's happening behind the scene> it downloads all the font files and applies it. but `.eot` is not supported so showed that warning and the rest first font is applied.

Reply:
For the sake of clarity just try to remove the `.eot` links and then you could verify the error would be gone but we don't want to remove this font for the sake of multiple browser support.